packages/aws/src/lib/CostAndUsageTypes.ts

```diff
@@ -169,6 +174,12 @@ export const UNKNOWN_USAGE_TYPES: string[] = [
   'Kafka.mcu.general',
   'SnapshotArchiveStorage',
   'PaidPrivateCA',
+  'Firehose-VpcDelivery-Hours',
+  'Airflow-MediumEnvironment',
+  'IPAddressManager-IP-Hours',
+  'Gateway:VTL-Storage',
+  'Aurora:ServerlessV2Usage', // RDS Aurora
+
```

Comment:
Super minor -- but there's an extra line here that is causing the Prettier check to fail and can be removed.
```suggestion

```